crm_rma_lot_mass_return/static/src/js/barcode_text.js

```diff
@@ -0,0 +1,110 @@
+openerp.crm_rma_lot_mass_return = function(openerp) {
```

Comment:
This widget need to be explained in the documentation.

It is necessary to load quickly by several criterias the information.